lib/QubitCsvTransform.class.php

```diff
@@ -99,27 +84,26 @@ public function writeHeadersOnFirstPass()
 
     public function initializeMySQLtemp()
     {
-        // Possible future cleanup: use QubitPdo (might have to add a method to set QubitPdo's private $conn property)
-        if (false === $link = mysqli_connect(getenv('MYSQL_HOST'), getenv('MYSQL_USER'), getenv('MYSQL_PASSWORD'), getenv('MYSQL_DB'))) {
-            throw new sfException('MySQL connection failed.');
-        }
-
-        $this->link = $link;
-
-        $sql = 'CREATE TABLE IF NOT EXISTS import_descriptions (
+        // Create transformation work area if it doesn't yet exist
+        $sql = 'CREATE TABLE IF NOT EXISTS '.self::WORKING_AREA_TABLE.' (
             id INT NOT NULL AUTO_INCREMENT,
             sortorder INT,
             data LONGTEXT,
             PRIMARY KEY (id))';
 
-        if (false === mysqli_query($link, $sql)) {
-            throw new sfException('MySQL create table failed.');
-        }
+        QubitPdo::modify($sql);
 
-        $sql = 'DELETE FROM import_descriptions';
-        if (false === mysqli_query($link, $sql)) {
-            throw new sfException('MySQL delete from import_descriptions failed.');
-        }
+        // Delete contents of work area
+        $sql = 'DELETE FROM '.self::WORKING_AREA_TABLE;
+
+        QubitPdo::prepareAndExecute($sql);
+    }
+
+    public static function dropMySQLtemp()
```

Comment:
I wonder if it's worth adding a class destructor that calls this method, in case the script is halted unexpectedly?

Reply:
Good thinking! I'll add that.